mimesis/providers/address.py

```diff
@@ -132,7 +132,7 @@ def postal_code(self) -> str:
         """
 
         mask = self._data['postal_code_fmt']
-        return custom_code(mask=mask)
+        return custom_code(mask=mask, rnd=self.random)
```

Comment:
It' better to move `custom_code` to  `CustomCodeMixin` and inherit it in `BaseProvider`.